packages/api/endpoints/granules.js

```diff
@@ -164,7 +169,18 @@ async function del(req, res) {
 
   // If the granule does not exist in PG, just log it
   try {
-    pgGranule = await granulePgModel.get(knex, { granule_id: granuleId });
+    if (dynamoGranule.collectionId) {
```

Comment:
Nit: 

----
If we ask to delete a granule from Dynamo that does *not* have a collectionId, pgGranule will be undefined despite there being a granule here....  oh I see, `deleteGranuleAndFiles` handles null for that. 

It might be helpful to expand the block  comment explaining that logic/reasoning briefly so as to avoid the above thought cycle, but it's just a nit.

Reply:
6d2a6e0b23f2ef4df19af7a12cee888e84efa6ec